sci-libs/openblas/openblas2-0.2.20.recipe

```diff
@@ -25,12 +30,15 @@ REQUIRES="
 	"
 
 PROVIDES_devel="
-	openblas${secondaryArchSuffix}_devel = $portVersion
-	devel:libopenblas$secondaryArchSuffix = $portVersion compat >= 0
-	devel:libopenblasp_r${portVersion}$secondaryArchSuffix = $portVersion compat >= 0
+	openblas2${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libopenblas$secondaryArchSuffix = $libVersionCompat
+	devel:libopenblasp_r$portVersion$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
-	openblas$secondaryArchSuffix == $portVersion base
+	openblas2$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	openblas3${secondaryArchSuffix}_devel
```

Comment:
I might be wrong, but it looks like both `openblas2-0.2.20.recipe` and `openblas3-0.3.1.recipe` provide `lib:libopenblas$secondaryArchSuffix`, so this means their base packages will overlap.

Now, if we tweak these recipes to allow these 2 base packages to co-exist, then I'd recommend a variant for the `CONFLICTS_devel`:

First, a small remark; Although we can :
* put a :
```
CONFLICTS_devel="
	openblas3${secondaryArchSuffix}_devel
	"
```
in  `openblas2-0.2.20.recipe`
* and, at the same time, put a :
```
CONFLICTS_devel="
	openblas2${secondaryArchSuffix}_devel
	"
```
in `openblas3-0.3.1.recipe`

it is enough to have only one `CONFLICTS_devel`, in any of these recipes.
But since the first `0.3.x` was released after the first `0.2.x`, I'd recommend keeping only the  `CONFLICTS_devel` in `openblas3-0.3.1.recipe`.